src/models/JobApplication/Errors/OfferNotTargetedForApplicantError.ts

```diff
@@ -0,0 +1,11 @@
+import { ApplicantType } from "$models/Offer";
+
+export class OfferNotTargetedForApplicantError extends Error {
+  public static buildMessage(applicantType: ApplicantType, targetApplicantType: ApplicantType) {
+    return `Applicant of type ${applicantType} cannot apply to offer targeted for ${targetApplicantType}`;
```

Comment:
no es "targeted for students" es "targeted to students"

Reply:
`A ${applicantType} cannot apply to offer for ${targetApplicantType}`